phase1-coordinator/src/coordinator_state.rs

```diff
@@ -2095,6 +2123,24 @@ impl CoordinatorState {
             }));
         }
 
+        // Update the IP map.
```

Comment:
Happy to add some more detail here 👍 

> For every IP address containing the participant being dropped, it will remove the IP if there are no other participants associated with that IP, or if there are other participants for the same IP it will just remove the dropped participant from being associated with the IP?

Correct!